plugins/generator-1.16.4/forge-1.16.4/procedures/world_data_current_time.java.ftl

```diff
@@ -0,0 +1 @@
+(world.getWorld().getDayTime())
```

Comment:
Use world info as in previous PRs please.

Reply:
Fixed!